packages/elements/list/src/transforms/marks/clearNodeMarks.tsx

```diff
@@ -0,0 +1,31 @@
+import { setNodes } from '@udecode/plate-common';
+import { TEditor, TNode } from '@udecode/plate-core';
+import { Path } from 'slate';
+import { PreviousStates } from '../../types';
+
+export const clearNodeMarks = (
```

Comment:
This is used by removeListItemContentMark, it is just a helper for that, maybe we shouldn't even export it...
Check normalizeListItemContentStyles (new name: normalizeListItemMarks) it uses this dirty flag...
Since this is only cleaes thing for the given types and does not change other I do not think an `unsetNodes` would suffice.